cuegui/cuegui/PreviewWidget.py

```diff
@@ -138,12 +158,12 @@ def __findHttpPort(self):
                     counter += 1
                     if counter >= 5000:
                         break
-                    if line.startswith("Preview Server"):
-                        return int(line.split(":")[1].strip())
+                    if "Preview Server" in line[:30]:
+                        return int(line.split(":")[-1].strip())
             finally:
                 fp.close()
 
-        raise Exception("Katana 2.7.19 and above is required for preview feature.")
+        raise Exception("This frame doesn't support previews. No Preview Server Found.")
```

Comment:
`Found` => `found`

`raise Exception("This frame doesn't support previews. No 'Preview Server' found.")`